pgx/src/pgx.ml

```diff
@@ -928,9 +928,9 @@ module Make (Thread : IO) = struct
       >>| function
       | [ ]
       | [ [ ] ] -> ()
-      | results -> fail_msg !"Pgx.execute_unit: Query returned a non-empty result but \
-                              execute_unit was expecting no result and found '%{sexp:row list}'"
-                     results
+      | results -> fail_msg "Pgx.execute_unit: Query returned a non-empty result but \
+                             execute_unit was expecting no result and found '%a'"
+                   Sexplib0.Sexp.pp ([%sexp_of: row list] results)
```

Comment:
Why do we need these changes? I think we strongly prefer the `!""` strings

Reply:
You're right, `ppx_custom_printf` is perfectly fine; I though it pulled in a `base` dependency, but that's only for the ppx itself.  Now updated to keep this and another custom printf.